src/citrine/informatics/generative_result.py

```diff
@@ -0,0 +1,57 @@
+from typing import List
```

Comment:
I suggest we rename this file to `generative_design.py` since it contains generally data classes for this end point, not just the result.